src/lib/exec-promise.js

```diff
@@ -4,23 +4,23 @@ const log = require('../lib/log');
 // TODO might be important to sanitise input to `exec()` to prevent injection
 // attacks by devious tech leads.
 
-module.exports = (...args) => new Promise((resolve, reject) => {
+module.exports = (log_level, ...args) => new Promise((resolve, reject) => {
 
     // Include stdout at log.LEVEL_WARN because xls2xform outputs warnings on stdout
 
-    const stdio = log.level >= log.LEVEL_WARN  ? [ 'ignore',  'pipe',   'pipe'  ] :
-                  log.level >= log.LEVEL_ERROR ? [ 'ignore', 'ignore',  'pipe'  ] :
+    const stdio = log_level >= log.LEVEL_WARN  ? [ 'ignore',  'pipe',   'pipe'  ] :
+                  log_level >= log.LEVEL_ERROR ? [ 'ignore', 'ignore',  'pipe'  ] :
```

Comment:
You didn't write this but while you're in here changing things, let's fix the nested ternary, eg: use `if/else`.

Reply:
Fixed